addons/account/views/report_invoice.xml

```diff
@@ -113,7 +114,7 @@
                                     <th name="th_discount" t-if="display_discount" t-attf-class="text-end {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}">
                                         <span>Disc.%</span>
                                     </th>
-                                    <th name="th_taxes" t-attf-class="text-start {{ 'd-none d-md-table-cell' if report_type == 'html' else '' }}"><span>Taxes</span></th>
```

Comment:
do we really want to remove those check ? I think this check is useful for the preview on the portal

Reply:
Still a missing d- ?